tests/Unit/Domain/FunctionsOfTime/Test_SettleToConstant.cpp

```diff
@@ -13,7 +13,7 @@
 #include "tests/Unit/TestHelpers.hpp"
 
 namespace {
-void test(const domain::FunctionsOfTime::FunctionOfTime& f_of_t,
+void test(const std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime> f_of_t,
```

Comment:
wait... is `f_of_t` used in this test function? How does this work without replacing a bunch of `.`s with `->`? (.. and why do the tests pass without that change?)

Reply:
I can bring #1856 into this PR. probably would be better.

Reply:
Ah, I see now that it's corrected in that commit. Yeah, I think it would be better to get the tests all sorted in a single PR.